src/middleware/set-run-date.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (req, res, next) => {
```

Comment:
nice addition of an express middleware to solve this problem and handle errors in the headers